pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -260,15 +260,50 @@ def test_to_datetime_tz_pytz(self):
                                  dtype='datetime64[ns, UTC]', freq=None)
         tm.assert_index_equal(result, expected)
 
-    def test_to_datetime_utc_is_true(self):
-        # See gh-11934
-        start = pd.Timestamp('2014-01-01', tz='utc')
-        end = pd.Timestamp('2014-01-03', tz='utc')
-        date_range = pd.bdate_range(start, end)
-
-        result = pd.to_datetime(date_range, utc=True)
-        expected = pd.DatetimeIndex(data=date_range)
-        tm.assert_index_equal(result, expected)
+    @pytest.mark.parametrize("init_constructor, end_constructor, test_method",
+                             [(Index, DatetimeIndex, tm.assert_index_equal),
+                              (list, DatetimeIndex, tm.assert_index_equal),
+                              (np.array, DatetimeIndex, tm.assert_index_equal),
+                              (Series, Series, tm.assert_series_equal)])
+    def test_to_datetime_utc_true(self,
+                                  init_constructor,
+                                  end_constructor,
+                                  test_method):
+        # See gh-11934 & gh-6415
+        data = ['20100102 121314', '20100102 121315']
+        expected_data = [pd.Timestamp('2010-01-02 12:13:14', tz='utc'),
+                         pd.Timestamp('2010-01-02 12:13:15', tz='utc')]
+
+        result = pd.to_datetime(init_constructor(data),
+                                format='%Y%m%d %H%M%S',
+                                utc=True)
+        expected = end_constructor(expected_data)
+        test_method(result, expected)
+
+        # Test scalar case as well
+        for scalar, expected in zip(data, expected_data):
+            result = pd.to_datetime(scalar, format='%Y%m%d %H%M%S', utc=True)
+            assert result == expected
+
+    def test_to_datetime_utc_true_with_series_single_value(self):
+        # GH 15760 UTC=True with Series
+        ts = 1.5e18
+        result = pd.to_datetime(pd.Series([ts]), utc=True)
+        expected = pd.Series([pd.Timestamp(ts, tz='utc')])
+        tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize("data, dtype",
+                             [('2013-01-01 00:00:00-01:00', None),
+                              ('2013-01-01 00:00:00-01:00', 'datetime64[ns]'),
+                              ('2013-01-01 01:00:00', None),
+                              ('2013-01-01 01:00:00', 'datetime64[ns]')])
```

Comment:
You can write this a bit shorter by doing:

```
@pytest.mark.parametrize("data", ['2013-01-01 00:00:00-01:00', '2013-01-01 00:00:00-01:00'])
@pytest.mark.parametrize("dtype", [None, 'datetime64[ns]'])
def test...
```

then you get all combinations of the two automatically.

But I still don't fully understand the point of the test (or for the '2013-01-01 01:00:00' case. Isn't that covered by the test above?)

What is not tested above is a string that included tz offset, or a series with an already datetime64 dtype (both aware or naive) So I would just test those two cases? (and both also seem distinct, so would put them in different tests)